Apigee/SmartDocs/Security/Oauth2Scheme.php

```diff
@@ -165,6 +171,26 @@ public function setAccessTokenParamName($name)
     }
 
     /**
+     * Get the in (location) value of the scheme.
+     *
+     * @return string
+     */
+    public function getIn()
+    {
+      return $this->in;
```

Comment:
@ossobuffo Indentation seems to be four spaces in netbeans, do you find it different in your editor?